modules/dashboard/README.md

```diff
@@ -0,0 +1,32 @@
+# Dashboard
+
+## Purpose
+
+The Dashboard serves as a landing page for all users logging into LORIS.  It provides introductory information and a primary quick-access interface for user workflows, including outstanding tasks and notifications of recent activity/additions in LORIS.  
+
+The Dashboard also provides at-a-glance live visual summaries of the data stored in LORIS, and dynamically plots data breakdowns that can be used in reporting. 
+
+## Intended Users
+
+All LORIS users can view the dashboard.  
+
+## Scope
+
+The Dashboard provides entrypoints to various workflows, but does not provide any tools for directly modifying data in the dashboard.  
+
+The Study Progression chart views at bottom may or may not count `excluded` candidates, and this should be clearly verified and documented/presented for the use of your LORIS users. 
+
+## Permissions
+
+Components of the Dashboard are governed by permissions related to module-specific privileges. For example, the _Document Repository_ notifications panel is displayed if a user has been granted either the `document_repository_view` or `document_repository_delete` permission. Site access privileges are also enforced. 
```

Comment:
```suggestion
Components of the Dashboard are governed by permissions related to module-specific privileges. For example, the _Document Repository_ notifications panel is displayed if a user has been granted either the `document_repository_view` or `document_repository_delete` permission. Site access privileges are also enforced. 
```

> Site access privileges are also enforced.

I don't think this is actually true right now. @racostas made a PR to address this but it's not active

Reply:
good catch, I'll tweak.  not sure what the suggestion is you're offering (github suggest syntax snafu?) but i'll make an edit and request your re-review.

Reply:
Oops yeah didn't mean to use the suggest feature

Reply:
np, Ready for your re-review.  i also added project permissions (because 22) and a mention under Scope.